Marlin/src/HAL/STM32/tft/tft_fsmc.h

```diff
@@ -121,7 +126,11 @@ const PinMap PinMap_FSMC_CS[] = {
   {NC,    NP,    0}
 };
 
-#define FSMC_RS(A)  (void *)((2 << A) - 2)
+#if ENABLED(TFT_INTERFACE_FSMC_8BIT)
+  #define FSMC_RS(A)  (void *)((2 << (A-1)) - 1)
```

Comment:
I confirmed that we need this. The address of the SRAM is `0x60000000U` + an offset based in the RS pin used. For PD_7, for example, is 0x20000 = 2^16. But, using 16 bit, it's 0x10000.
So, this code is right for 8 bit.